src/package.xml

```diff
@@ -3936,6 +3957,7 @@
     <types>
         <members>CumulusStaticResources</members>
         <members>DefaultGiftEntryTemplate</members>
+        <members>gsResources</members>
```

Comment:
I recommend spelling out the name of the static resource. I believe this will require renaming the file and folder under StaticResourceSources too.
```suggestion
        <members>GettingStartedResources</members>
```